WooCommerce/Classes/Authentication/Epilogue/StorePickerCoordinator.swift

```diff
@@ -123,6 +123,12 @@ private extension StorePickerCoordinator {
         }
         ServiceLocator.stores.dispatch(orderAction)
 
+        group.enter()
+        let reviewAction = ProductReviewAction.resetStoredProductReviews {
+            group.leave()
+        }
+        ServiceLocator.stores.dispatch(reviewAction)
+
```

Comment:
I don't think we want this. If we leave the cached data alone, loading a previous Site will be a faster experience for the user. But I digress.

Reply:
Maybe you are not digressing... I added that to match the cleanup that is going on with other model objects, but given the recent discussions, it might not make sense. In any case, are you okay with leaving as is now (clearing the cached data) until we make a general decision about what to do with cached data when switching stores / logging out?